tests/cypress/e2e/englishTranslation.cy.ts

```diff
@@ -8,7 +8,7 @@ describe('English translation', { testIsolation: false }, () => {
     .forEach((device) => {
       context(device, Cypress.env('resolution')[`${device}`], () => {
         before(() => {
-          cy.visit('/kontakty')
+          cy.visit('/kontakty-a-uradne-hodiny')
```

Comment:
We chould revriet the english translation test to run on homepage.
- visit SK homepage
- click on English in NavBar
- wait to visit EN homepage, check the main heading (Welcome...)
- click to Slovak in NavBar
- wait to visit SK homepage, check the main heading (Vitajte...)

Even tho the main heading is from Strapi, it hasn§t been changed from the beginning. So we can rely on in quite well. Better than on Contacts page.